sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -4748,7 +4748,7 @@ class ResourceDetails:
 
     custom_document_models: CustomDocumentModelsDetails
     """Details regarding the custom models under the Form Recognizer resource."""
-    neural_document_model_quota: QuotaDetails
+    neural_document_model_quota: Optional[QuotaDetails]
```

Comment:
This field is required in the new version but didnt exist in the previous GA so marking as optional